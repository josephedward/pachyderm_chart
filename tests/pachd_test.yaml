templates:
  - pachd-deployment.yaml
  - pachd-deployment.yaml
  - pachd-peer-deployment.yaml
tests:
  # or you can snapshot the whole manifest
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}