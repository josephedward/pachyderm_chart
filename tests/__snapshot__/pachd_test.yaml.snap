manifest should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: pach_chart
        app.kubernetes.io/version: 0.5.48
        helm.sh/chart: pach_chart-0.1.0
      name: RELEASE-NAME-pach_chart
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: pach_chart
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: pach_chart
        spec:
          containers:
          - image: pachyderm/pachd:1.11.7
            imagePullPolicy: IfNotPresent
            name: pach_chart-pachd
            ports:
            - containerPort: 650
              name: api-grpc-port
              protocol: TCP
            - containerPort: 651
              name: trace-port
              protocol: TCP
            - containerPort: 652
              name: api-http-port
              protocol: TCP
            - containerPort: 653
              name: peer-port
              protocol: TCP
            - containerPort: 655
              name: api-git-port
              protocol: TCP
            - containerPort: 654
              name: saml-port
              protocol: TCP
            - containerPort: 657
              name: oidc-port
              protocol: TCP
            resources: {}
            securityContext: {}
          securityContext: {}
          serviceAccountName: RELEASE-NAME-pach_chart
  2: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: pach_chart
        app.kubernetes.io/version: 0.5.48
        helm.sh/chart: pach_chart-0.1.0
      name: RELEASE-NAME-pach_chart
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: pach_chart
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: pach_chart
        spec:
          containers:
          - image: pachyderm/pachd:1.11.7
            imagePullPolicy: IfNotPresent
            name: pach_chart-pachd
            ports:
            - containerPort: 650
              name: api-grpc-port
              protocol: TCP
            - containerPort: 651
              name: trace-port
              protocol: TCP
            - containerPort: 652
              name: api-http-port
              protocol: TCP
            - containerPort: 653
              name: peer-port
              protocol: TCP
            - containerPort: 655
              name: api-git-port
              protocol: TCP
            - containerPort: 654
              name: saml-port
              protocol: TCP
            - containerPort: 657
              name: oidc-port
              protocol: TCP
            resources: {}
            securityContext: {}
          securityContext: {}
          serviceAccountName: RELEASE-NAME-pach_chart
  3: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: pach_chart
        app.kubernetes.io/version: 0.5.48
        helm.sh/chart: pach_chart-0.1.0
      name: RELEASE-NAME-pach_chart
    spec:
      ports:
      - name: api-grpc-peer-port
        port: 30653
        targetPort: 653
      selector:
        app: pachd
      type: ClusterIP
